src/comm/USBBoardInfo.json

```diff
@@ -17,6 +17,7 @@
         { "vendorID": 7052, "productID": 54,        "boardClass": "Pixhawk",    "name": "PX4 FMU V6U" },
         { "vendorID": 12677, "productID": 53,       "boardClass": "Pixhawk",    "name": "PX4 FMU V6X" },
         { "vendorID": 12677, "productID": 56,       "boardClass": "Pixhawk",    "name": "PX4 FMU V6C" },
+        { "vendorID": 3643, "productID": 29,        "boardClass": "Pixhawk",    "name": "PX4 FMU V6X-RT" },
```

Comment:
@mrpollo I think theses are all in decimal.

The number in the email were in HEX.  Both need to be converted to decimal.

Reply:
Good catch! Thanks @davids5